src/applications/appeals/995/validations/date.js

```diff
@@ -1,56 +1,48 @@
 import moment from 'moment';
 
-import { parseISODate } from 'platform/forms-system/src/js/helpers';
-
-import { fixDateFormat } from '../../shared/utils/replace';
 import { errorMessages } from '../constants';
 
-import { FORMAT_YMD, MAX_YEARS_PAST } from '../../shared/constants';
+import { MAX_YEARS_PAST } from '../../shared/constants';
 
 import {
-  buildErrorParts,
-  createPartsError,
-  isInvalidDateString,
-  hasErrorParts,
+  createScreenReaderErrorMsg,
+  dateFunctions,
 } from '../../shared/validations/date';
 
 export const minDate = moment()
   .subtract(MAX_YEARS_PAST, 'year')
   .startOf('day');
 
-const maxDate = moment().startOf('day');
-
 export const validateDate = (errors, rawString = '', fullData) => {
```

Comment:
`rawDateString` updated but holding off on `validateDate` because it looks like it'll be a more invasive change as it's called 61 times across 13 files.